docs/accessControl.md

```diff
@@ -256,22 +253,20 @@ The above `allowAccess` rule is appropriate for an `infoCourseInstance.json` fil
 ```json
 "allowAccess": [
     {
-        "mode": "Exam",
-        "credit": 100,
-        "startDate": "2014-09-07T00:00:01",
-        "endDate": "2014-09-10T23:59:59"
-    },
-    {
-        "mode": "Exam",
         "uids": ["student1@example.com", "student2@example.com"],
         "credit": 100,
         "startDate": "2014-09-12T00:00:01",
         "endDate": "2014-09-12T23:59:59"
+    },
+    {
+        "credit": 100,
+        "startDate": "2014-09-07T00:00:01",
+        "endDate": "2014-09-10T23:59:59"
     }
 ],
```

Comment:
I changed the order in this example mostly to stick to convention, where exceptions are listed first. However, in this example, the order didn't have a practical effect, since the time slots did not overlap.